cmd/istio-cni/main.go

```diff
@@ -219,7 +219,11 @@ func cmdAdd(args *skel.CmdArgs) error {
 							logger.Errorf("Pod redirect failed due to unavailable InterceptRuleMgr of type %s",
 								interceptRuleMgrType)
 						} else {
-							rulesMgr := interceptMgrCtor()
+//							rulesMgr := interceptMgrCtor()
+							// TODO Depending on a parameters, either iptables or nftables interface must be selected
+							// and instantiated, for now defaulting to iptables.
+							// TODO (sbezverk) WIP nftables, will switch back InterceptRuleMgr selection when it is ready
+							rulesMgr := newNFTables()
```

Comment:
The latest install-test CI failure is due to this line.  My change for the InterceptMgrCtor() stuff made the tests configure the mockInterceptRuleMgr via the plugin config.  You should remove your TODOs and change this line back to the `rulesMgr := interceptMgrCtor()` AND

1.  In `cmd/istio-cni/intercept_rule_mgr.go` add a intercept type config string mapping to the constructor newNFTables
1.  In your testing change the plugin config to have the setting "intercept_type" set to your nft config string.

Reply:
Fixed, hopefully